### ISSUE TITLE
Why is Person recognized as Organization?

### ISSUE THREAD

**Issue by monajalal** [explosion/spaCy#2330]
In the code below, why is 'Bernie' recognized as an organization and not as a person?

```
import spacy
nlp = spacy.load('en')
doc1 = nlp(u"Trump on the other hand goes all in on water boarding AND some.")
doc2 = nlp(u"RT @mtaibbi The hysterical concern over how to pay for Bernie's plans is hilarious. Nobody worries about how we afford the F-35. https://t.co/PhstjXx1Qx")
doc5 = nlp(u"Rami Eid is studying at Stony Brook University in New York")
for ent in doc2.ents:
	print(ent, ent.label, ent.label_)
```

result is:
```
RT @mtaibbi 381 ORG
Bernie 381 ORG
F-35 381 ORG
```

Also running it on doc1, 'Trump' is not recognized as an entity. Why is that?
Thanks.

**Comment by ines** [comment]
The process of recognising entities is *statistical* – the English model has no deeper "knowledge" of what a person or an organisation is, it only *predicts* what label is most likely in this context, based on the data it was trained on.

In the case of the English model, the training data was general news and web text. In your example, you're processing tweets, which are very different from "regular" text. For example, they contain URLs (which almost never appear in non-web text) and tokens like "RT @mtaibbi" which are very specific to Twitter, and they also don't always follow consistent grammatical rules, spelling or capitalisation.

In order to train a good model that can generalise well across a lot of text, you also need *a lot* of labelled training data. The data spaCy's English models are trained on is considered pretty "modern" – but it's still over 5 years old. "Trump" and "Bernie" are actually a great examples of why this matters – five years ago, the entity "Trump" was much less commonly used, and very often also referring to the Trump Organization. People also weren't talking about "Bernie" much, so based on the context ("pay for XXX's plans"), the model guesses that this term is more likely a company than a person. 

**TL;DR:** If you train a model on news text and process tweets, you likely won't see good results – just like a model trained on Twitter likely won't perform well on medical text etc. That's why you almost *always* want to adjust the model to your specific domain if extracting named entities is important to you. You can find more details on this in the [documentation on training and updating models](https://spacy.io/usage/training).
